lamby/test/status_test.py

```diff
@@ -0,0 +1,69 @@
+from src.status import status
+from src.init import init
+from src.commit import commit
+from src.uninit import uninit
+from src.checkout import checkout
+from src.utils import (
+    deserialize_log
+)
+from test.utils import (
+    create_file
+)
+
+
+def test_status_basic(runner):
```

Comment:
these are technically integration tests since you're invoking other dependent methods to test functionality; make at least one test that tests status without depending on init and commit first